app/tests/e2e/test_messaging.py

```diff
@@ -17,6 +18,8 @@ async def test_send_trust_ping(
         connection_id=bob_and_alice_connection["alice_connection_id"], comment="Donda"
     )
 
+    # Wait for connection to be ready - sometimes this takes a second or two
+    sleep(5)
```

Comment:
Maybe we can add an event listener to the bob_and_alice_connection so we don't have to add timeouts throughout the codebase? That way it's fixed once for all tests that use `bob_and_alice_connection`